### PR TITLE
Update community page

### PR DESCRIPTION
**Pull request by FloorD**
Following #465, I've added some styling, and a first version of an event list (to be automated), to have something to look at. The Community page was very bare-bones before. 

A screenshot of what I did:

screencapture-localhost-8000-docs-community-html-2022-01-11-15_13_43

### CODE REVIEW

File: docs/community.rst [hunk 1]
```diff
@@ -3,13 +3,83 @@ Aiven community
...
+
+January 20, `PHP User Group Frankfurt <https://www.phpugffm.de/veranstaltungen-2/phpugffm-i-2022/>`_
+
+Join Lorna Jane for "Aiven for Laravel" 
+
+------------
+
+February 5-6, `FOSDEM <https://fosdem.org/2022/>`_
+
+Join Francesco Tisiot and Olena Kutsenko for their sessions
```
Comment (laysauchoa) [inline docs/community.rst:71]: instead of names, I would suggest to add the github handle

Comment (TibsAtWork) [reply docs/community.rst:71]: Or both? Since the casual reader will not necessarily realise who the github id means.

Comment (ftisiot) [reply docs/community.rst:71]: Why not the Twitter handle?

Comment (laysauchoa) [reply docs/community.rst:71]: I think is like, most likely people want to contact you somehow and from github handle is more like looking what you are doing and github people can drop your message. Another option, it is e-mail. Up to the team. I'd be fine either way. :)

Comment (TibsAtWork) [reply docs/community.rst:71]: And yes, comparing that to what is there now, that is much nicer.

**Comment by TibsAtWork** [comment]
And yes, comparing that to the previous page, it is much nicer.

**Comment by anelook** [comment]
Cool info and give some energy to the page. 
How will we maintain this list?

**Comment by FloorD** [comment]
> Cool info and give some energy to the page. How will we maintain this list?

The idea would be to hook an API in here. So maybe pulling from the Google calendar (and follow conventions like putting the speaker and talk title in the descr. of the calendar item), or from a (to be created) Meetup group. Until that's decided I'd be happy to maintain this. 

We can link names to Twitter accounts or GH (instead of just using usernames). Would need to figure out how we can get that to work in an automated workflow but should be possible.

**Comment by laysauchoa** [comment]
Why do you think we need some external automation for the twitter names? Do you think something like this -> [@laysauchoa](https://twitter.com/laysauchoa) would work for this case? I have not found of twitter account can be modified, so I guess they cannot.

**Comment by anelook** [comment]
Alternatively, a simple solution can be to keep up-to-date info in a google spreadsheet and inject it here as an iframe. (just got this idea from Felix @runwuf :D)

**Comment by runwuf** [comment]
static google spreadsheet can be easily integrated by embed the code generated out of `File > Share > Publish to web` -> `Embed` -> `Publish`, but it is going to be static, to get a live/editable spreadsheet embed is a bit tricky, ping me if needed. I might document the steps after.

**Comment by lornajane** [comment]
The styling improvements by themselves are worth it :) I'm not sure about maintaining a list of events by hand but if you want to do that then we can try, and just remove the section if it doesn't work out!

The build failed because it doesn't know our proper names (also can you use "Lorna Mitchell" for me please?), you can add them to the vocabulary file (info is in the readme) to permit those words.

**Comment by laysauchoa** [comment]
rebase with the main page

**Comment by laysauchoa** [comment]
The way that it is, you won't be redirected to the right links in none of the buttons. I had the same issue and one needs to remove the ` :type: ref`. Leaving here for your reference.

```
    .. link-button:: https://twitter.com/aiven_io
        :type: ref
        :text: To the Twitterverse
        :classes: stretched-link
```

**Pull request closed without merging**